MultiSource/Benchmarks/Ptrdist/ks/KS-1.c

```diff
@@ -46,33 +47,37 @@ ReadNetList(char *fname : itype(_Ptr<char>))
 	exit(1));
 
     TRY(fgets(line, BUF_LEN, inFile),
-	sscanf(line, "%lu %lu", &numNets, &numModules) == 2, "ReadData",
+	sscanf((const char*)line, "%lu %lu", &numNets, &numModules) == 2, "ReadData",
 	"unable to parse header in file [%s]", inFile, 0, 0,
 	exit(1));
 
     for (net = 0; net < numNets; net++) {
 	fgets(line, BUF_LEN, inFile);
 	
 	/* net connections for "dest" */
-	dest = atol(strtok(line, " \t\n"))-1;
+	_Unchecked {dest = atol(strtok((char*)line, " \t\n"))-1;}
 
 	/* parse out all the net module connections */
 	TRY(head = prev = calloc(1, sizeof(Module)),
 	    prev != NULL, "ReadData",
 	    "unable to allocate a module list node", 0, 0, 0,
 	    exit(1));
-	(*prev).module = atol(strtok(NULL, " \t\n"))-1;
+	_Unchecked {(*prev).module = atol(strtok(NULL, " \t\n"))-1;}
 	(*prev).next = NULL;
+  _Unchecked {
+    char *tok;
 	while ((tok = strtok(NULL, " \t\n")) != NULL) {
+    _Checked {
```

Comment:
I would make this `_Checked` only wrap the `node = calloc(...)` argument, as the TRY macro has an _Unchecked wrapping all of what it expands to.